packages/dso-toolkit/src/components/card/card.scss

```diff
@@ -45,47 +45,6 @@
   }
 }
 
-dso-card {
-  .dso-card-heading a {
-    text-decoration: none;
-
-    dso-icon:last-child {
-      align-self: flex-start;
-    }
-  }
-
-  h2[slot="heading"],
-  h3[slot="heading"],
-  h4[slot="heading"],
-  h5[slot="heading"] {
```

Comment:
Deze selector is gefuseerd met `::slotted([slot="heading"])` ✅ 

Maar ik begrijp niet waar de styling uit de nested `span` is gebleven, kan je dat toelichten?

Reply:
die kan niet in `::slotted` om de bekende redenen, en de boel viel visueel niet om bij weglaten

Reply:
Maar heb je uitgezocht waarom?

Nu ik nog een keer naar de styling kijk verwacht ik onderstaande structuur:

```html
<element "slot=heading">
  <span>
  <dso-icon | svg class="di">
```

Maar dat zie ik niet terugkomen in de templates. Dus ik heb wat archeologische gitbedrijvingen uitgevoerd:
* Deze styling is in https://github.com/dso-toolkit/dso-toolkit/commit/d49d0434d4d19db7a1cab2bf939ec6723626acd8 geintroduceerd
* In https://github.com/dso-toolkit/dso-toolkit/commit/73a7ab93ac945111a64e953b10a8bcae0acd253c is de markup aangepast maar is de bijbehorende styling niet verwijderd.

De laatste change is als een BREAKING release uitgebracht. Ik denk dat we deze styling daarmee veilig kunnen verwijderen. Mocht er bij afnemers toch iets omvallen is de oplossing dat ze markup compliant moeten worden.

cc @iterox @cvanderleij-ilionx

Reply:
De markup in de html/css versie heeft deze styling *wel* nodig, maar als je die gelijktrekt, heb je meteen weer een breaking change..

Reply:
HTML/CSS:
- a
- -  h2
- --  span
- --  svg

CORE:
- a
- -  slot 'heading' (zonder kinderen)
- -  icon

Reply:
Akkoord, maar we hebben het nu specifiek over de Web Component implementatie. 

Deze thread begon omdat ik mij afvroeg waarom de styling uit de geneste span niet is overgezet naar het Web Component. Jij gaf aan dat er visueel niets omviel, ik heb uitgezocht waarom dat zo is. Dus volgens mij zijn we eruit en kan deze thread worden opgelost.

Reply:
als we de styling weghalen valt de html/css implementatie om, komt de pijl dichter bij de tekst te staan. dat los je alleen op door de markup aan te passen, wat een breaking change oplevert. dus wat doen we: laten we de styling staan, of passen we de markup aan?

Reply:
Telefonisch afgestemd, we begrijpen elkaar :)